corehq/messaging/smsbackends/telerivet/views.py

```diff
@@ -51,6 +53,34 @@ def incoming_message(request):
     return HttpResponse()
 
 
+@waf_allow('XSS_BODY')
+@require_POST
+@csrf_exempt
+def message_status(request, message_id):
+    status = request.POST.get('status')
+    logger.info(f'Updating Telerivet message status: message_id={message_id}, status={status}')
```

Comment:
do you need the logging now? I guess you added it for debugging on staging.

Reply:
Mostly yes, but I thought it could be nice to have for the future. I think I'll rather remove it since we don't really seem to do logging as a rule...

Reply:
> I thought it could be nice to have for the future

That could be true. The logs are cycled right? so we would lose these. and I am hoping you or anyone would need to debug this on production (considering your comment in the description)

> we don't really seem to do logging as a rule

True.

Reply:
are we keeping these?

Reply:
@mkangia I want to keep it at least for the duration of QA on production. They have successfully tested on staging, but as far as I know they plan to test this again once deployed. I want to keep the log in case something weird happens and I need to see what Telerivet sends back or does not send...I will make a note to remove the log once QA is done on prod.